python/lsst/jointcal/jointcal.py

```diff
@@ -405,7 +405,7 @@ class JointcalInputData:
     """The WCS of this exposure."""
     bbox: lsst.geom.Box2I
     """The bounding box of this exposure."""
-    filter: lsst.afw.image.Filter
+    filterLabel: lsst.afw.image.FilterLabel
```

Comment:
I thought the consensus on `#dm-naming-things` was that we wanted "filter" to be the final API. If you don't need `filter` and `filterLabel` to coexist for backwards compatibility, why not use the former right away?